src/wazuh_modules/vulnerability_detector/wm_vuln_detector.c

```diff
@@ -174,6 +174,14 @@ STATIC int wm_vuldet_sync_feed(update_node *upd, int8_t *need_update);
 STATIC int wm_vuldet_check_feed(update_node *upd, int8_t *updated);
 STATIC vu_feed wm_vuldet_set_allowed_feed(const char *os_name, const char *os_version, update_node **updates, vu_feed *agent_dist);
 
+/**
+ * @brief Gets the last update time of an specific feed from the metadata database.
+ *
+ * @param feed The feedfrom which to obtain the information.
```

Comment:
Typo: missing space after `feed`.
Also, we can consider a different expression for this sentence.